timecodexml2webvtt.cpp

```diff
@@ -95,7 +95,7 @@ int main(int argc, char* argv[])
     }
     input_file.seekg(0);
     auto input = new char[input_size];
-    if (!input_file.read(input, input_size).eof()) {
+    if (input_file.read(input, input_size).fail()) {
```

Comment:
But works on Windows only if `ios::binary` is used while opening the file, looks like that the count of characters to read is not the count of bytes, `\r\n` counting for 1 char.
And we don't have the count of chars to read, only the file size.

So better to have this change + `ios::binary` during opening (and the XML parser handles `\r\n`).